addons/hr_timesheet/models/project.py

```diff
@@ -80,18 +80,26 @@ def _search_is_internal_project(self, operator, value):
             operator_new = 'not inselect'
         return [('id', operator_new, (query, ()))]
 
-    @api.depends('allow_timesheets', 'task_ids.planned_hours', 'task_ids.remaining_hours')
+    @api.depends('allow_timesheets', 'task_ids.planned_hours', 'timesheet_ids')
     def _compute_remaining_hours(self):
-        group_read = self.env['project.task'].read_group(
-            domain=[('planned_hours', '!=', False), ('project_id', 'in', self.filtered('allow_timesheets').ids),
+        group_read_effective_hours = self.env['account.analytic.line'].read_group(
+            domain=[('project_id', 'in', self.filtered('allow_timesheets').ids),
+                    ('task_id', '!=', False), ('task_id.planned_hours', '!=', 0.0),
+                    '|', ('task_id.stage_id.fold', '=', False), ('task_id.stage_id', '=', False)],
+            fields=['effective_hours:sum(unit_amount)'], groupby='project_id')
+        group_read_planned_hours = self.env['project.task'].read_group(
+            domain=[('planned_hours', '!=', 0.0), ('project_id', 'in', self.filtered('allow_timesheets').ids),
                      '|', ('stage_id.fold', '=', False), ('stage_id', '=', False)],
-            fields=['planned_hours:sum', 'remaining_hours:sum'], groupby='project_id')
-        group_per_project_id = {group['project_id'][0]: group for group in group_read}
+            fields=['planned_hours:sum', ], groupby='project_id')
+        group_planned_hours_per_project_id = {group['project_id'][0]: group for group in group_read_planned_hours}
```

Comment:
Why not just having the planned hours as values in the dict. 🤔 
```suggestion
        planned_hours_per_project_id = {res['project_id'][0]: res['planned_hours'] for res in task_read_group}
```